cypress/e2e/cloud/operator.test.ts

```diff
@@ -7,6 +7,7 @@ describe('Operator Page', () => {
           cy.setFeatureFlags({
             uiUnificationFlag: true,
           }).then(() => {
+            cy.initializeQuartzMockOperator()
```

Comment:
I might add a comment in this test as well to flag that `quartzMockOperator` is using an endpoint that doesn't exist for the UI in prod.

Reply:
Good thinking, I'll add the comment!